src/Tribe/Service_Providers/Tooltip.php

```diff
@@ -32,22 +32,22 @@ private function hook() {
 	/**
 	 * Register assets associated with tooltip
 	 *
-	 * @since 4.9.8
```

Comment:
I think this shouldn't be modified if the function is already there. For the rest is good to go!